sevenbridges/models/file.py

```diff
@@ -154,8 +156,13 @@ def download(self, path, retry=5, timeout=10,
         :param timeout:  Timeout for http requests.
         :param chunk_size:  Chunk size in bytes.
         :param wait: If true will wait for download to complete.
+        :param over_write: If True will silently overwrite existing file, otherwise OSError is raised
         :return: Download handle.
         """
+
+        if not over_write and os.path.exists(path):
+            raise OSError(17, 'File exists')
```

Comment:
@kghosesbg We should always raise our own errors. Please see the errors.py file. It should subclass the SbgError. Another thing is the param name `over_write`, I call it overwrite everywhere else can you please change it so it reflects that.